From 2335302b569eec0b5397595295382fa98e9d0f9a Mon Sep 17 00:00:00 2001
From: Ishimoto Shinobu <nagakamira@gmail.com>
Date: Sat, 27 Mar 2021 11:39:40 +0900
Subject: [PATCH] thumbnail: Fix sandboxing on MUSL systems

Don't force bind /etc/ld.so.cache, as it isn't used on non-glibc systems
such as MUSL-based OSes.

Closes: #117
---
 libgnome-desktop/gnome-desktop-thumbnail-script.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libgnome-desktop/gnome-desktop-thumbnail-script.c b/libgnome-desktop/gnome-desktop-thumbnail-script.c
index a21597a9..ddcc1511 100644
--- a/libgnome-desktop/gnome-desktop-thumbnail-script.c
+++ b/libgnome-desktop/gnome-desktop-thumbnail-script.c
@@ -557,7 +557,7 @@ add_bwrap (GPtrArray   *array,
   add_args (array,
 	    "bwrap",
 	    "--ro-bind", "/usr", "/usr",
-	    "--ro-bind", "/etc/ld.so.cache", "/etc/ld.so.cache",
+	    "--ro-bind-try", "/etc/ld.so.cache", "/etc/ld.so.cache",
 	    NULL);
 
   /* These directories might be symlinks into /usr/... */
-- 
GitLab


